src/rules/no-unnormalized-keys.js

```diff
@@ -0,0 +1,54 @@
+/**
+ * @fileoverview Rule to detect unnormalized keys in JSON.
+ * @author Bradley Meck Farias
+ */
+
+export default {
+	meta: {
+		type: "problem",
```

Comment:
```suggestion
		type: /** @type {const} */ ("problem"),
```

To fix the Test Types failure.

(the same change was done in other rules in https://github.com/eslint/json/pull/65).